src/misc.asm

```diff
@@ -102,8 +102,8 @@ if !FEATURE_PAL
 org $90EA38
 else
 org $90EA3B
-    BRA $0B
 endif
+    BRA $0B
```

Comment:
Yikes!  How long did it take you to find this?

If the game is crashing when completing Ceres, I wonder why this would fix it.

Reply:
This change only fixes the pause check. I thought GitHub was playing tricks on me for a second there, but it didn't merge the Ceres elevator change because it was only missing on my branch. You must have fixed it already back when you were going over PAL differences.